app/controllers/authors_controller.rb

```diff
@@ -282,81 +282,63 @@ def by_tag
   end
 
   def create_toc
-    author = Person.find(params[:id])
-    unless author.nil?
```

Comment:
Actually we have many places like this all over the code. They are useless, as ActiveRecord.find method raises an RecordNotFound exception if record with given id is not found. So we never get author.nil? condition satisfied,  but code expects nil to be returned in such case.